static/js/admin/vue-permissions-editor/tests/unit/patronRoles.spec.js

```diff
@@ -535,6 +558,25 @@ describe('patronRoles.vue', () => {
         });
     });
 
+    it("does not merge display permissions if assigned roles are different but inherited roles are the same", (done) => {
```

Comment:
Should this test be verifying that the parents aren't merged? And that the correct rows are being marked as active? (both the parent roles I believe).

Reply:
Yes. I think I copied another test and apparently only partly updated it.